specification/storagemover/resource-manager/readme.python.md

```diff
@@ -3,21 +3,16 @@
 These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
-``` yaml $(track2)
+``` yaml $(python)
```

Comment:
Hi @msyyc , I'm wonderring shall we merge line 6~18 to same yaml block?

Reply:
we want keep same with [template file](https://github.com/Azure/azure-rest-api-specs/tree/main/documentation/samplefiles) and existing python.md